jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1424,7 +1427,7 @@
     (let (avs)
       (setq avs
             (list
-              (send *baxter* :approach-from-downside arm)
+              (send *baxter* :midpose-hold-opposite-hand-object-downside arm)
```

Comment:
> 英語の理解力がないんですがどこで再利用の話をしてたのでしょうか？

ここで、fold-pose-lowerが使いまわせないように、
ハードコードなメソッドはどうせ使いまわせないから、使いまわそうとしても無駄じゃないかという意味で言っています。
（だから使いまわせる段階になってからmasterにマージするか、）
使いまわすのは諦めて `:midpose-[使われるメソッド]` と書いたほうが読みやすいと言う考えです。
少し遠回しでしたね。

> For longterm use, I think we should use some planning algorithms with checking collisions with tote or some other objects,
as we checked norm distance between IK result and :avoid-shelf-pose-X in :move-arm-body->bin with multiple predefined poses.
Using a single hardcoded-pose works in very limited situations.
For example, we have fold-pose-lower, a predefined pose, but it can't be used for the :hold-opposite-hand-object because of collision with robot itself or tote, right?